contracts/src/v0.8/functions/tests/v1_X/testhelpers/FunctionsClientHarness.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {FunctionsClientUpgradeHelper} from "./FunctionsClientUpgradeHelper.sol";
+import {FunctionsResponse} from "../../../dev/v1_X/libraries/FunctionsResponse.sol";
+
+/// @title Functions Client Test Harness
+/// @notice Contract to expose internal functions for testing purposes
+contract FunctionsClientHarness is FunctionsClientUpgradeHelper {
```

Comment:
What is the reason for creating this new FunctionsClientHarness as opposed to just extending the FunctionsClientUpgradeHelper with these additional methods?

Reply:
`FunctionsClientUpgradeHelper` is a contract that is actually in use during the deployment process, it's the consumer contract that gets deployed to blue/green test deployments